persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbSampleRepository.java

```diff
@@ -58,12 +68,18 @@ MetadbSample findResearchSampleByRequestAndIgoId(@Param("reqId") String reqId,
 
     @Query("MATCH (pa: PatientAlias {namespace: 'cmoId', value: $cmoPatientId})-[:IS_ALIAS]->"
             + "(p: Patient)-[:HAS_SAMPLE]->(s: Sample)-[:HAS_METADATA]->(sm: SampleMetadata) "
-            + "MATCH (r: Request)-[:HAS_SAMPLE]->(s) SET sm.igoRequestId = r.igoRequestId "
-            + "RETURN sm"
-    )
-    List<SampleMetadata> findAllSampleMetadataListByCmoPatientId(@Param("cmoPatientId") String cmoPatientId);
+            + "RETURN sm")
+    List<SampleMetadata> findAllSampleMetadataByCmoPatientId(
+            @Param("cmoPatientId") String cmoPatientId);
 
-    @Query("MATCH (sa :SampleAlias {value: $igoId, namespace: 'igoId'})-[:IS_ALIAS]->(s: Sample)"
-            + "-[:HAS_METADATA]->(sm: SampleMetadata) RETURN sm")
+    @Query("MATCH (sa :SampleAlias {value:$igoId, namespace: 'igoId'})"
+            + "-[:IS_ALIAS]->(s: Sample)"
+            + "-[:HAS_METADATA]->(sm: SampleMetadata)"
+            + "RETURN sm")
     List<SampleMetadata> findResearchSampleMetadataHistoryByIgoId(@Param("igoId") String igoId);
+
+    //needs to be tested with clinical samples
+    @Query("MATCH (sa :SampleAlias {value: $dmpId, namespace: 'dmpId'})-[:IS_ALIAS]->(s: Sample)"
+            + "-[:HAS_METADATA]->(sm: SampleMetadata) RETURN sm")
+    List<SampleMetadata> findClinicalSampleMetadataHistoryByDmpId(@Param("dmpId") String dmpId);
```

Comment:
Same comment here regarding how we can prob consolidate these two methods with a single one

```
@Query("MATCH (sa :SampleAlias {value: $sampleAlias.value, namespace: $sampleAlias.namespace})-[:IS_ALIAS]->(s: Sample)"
        + "-[:HAS_METADATA]->(sm: SampleMetadata) RETURN sm")
List<SampleMetadata> findSampleMetadataHistoryByAlias(@Param("sampleAlias") SampleAlias sampleAlias);
```